include/wlr/util/edges.h

```diff
@@ -0,0 +1,12 @@
+#ifndef WLR_UTIL_EDGES_H
+#define WLR_UTIL_EDGES_H
+
+enum wlr_edges {
+	WLR_EDGE_NONE = 0,
+	WLR_EDGE_TOP = 1,
+	WLR_EDGE_BOTTOM = 2,
+	WLR_EDGE_LEFT = 4,
+	WLR_EDGE_RIGHT = 8,
+};
```

Comment:
This should go into xcursor instead of getting its own header.

Reply:
We discussed this in IRC and decided that since it's used in xwayland it shouldn't be (only) in xcursor (to keep xwayland and xcursor separate).

Reply:
Why would we need to keep xwayland and xcursor separate?

Reply:
Because we're building a modular Wayland library and `wlr_xcursor` is optional for compositors. If we put this enum in `wlr_xcursor`, compositors not using `wlr_xcursor` will still need to include it to get the enum for xwayland. `wlr_xcursor` and xwayland are unrelated.

Reply:
I don't like either solution. This is modularity to the point of absurdity. I guess we'll just leave it here.

Reply:
This is a consequence of not having a single shell abstraction. If we did, I think it would belong there because resize edges seems more like a shell concept to me. The only source of this data structure is a shell event.